askup/templates/askup/base.html

```diff
@@ -1,27 +1,29 @@
-<html><head>
-    <title>AskUp</title>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<html>
+    {% load static %}
+    <head>
+        <title>AskUp</title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
-    <script data-turbolinks-track="true" src="/static/assets/application.js"></script>
-    <link data-turbolinks-track="true" href="/static/assets/application.css" media="all" rel="stylesheet">
+        <script data-turbolinks-track="true" src="{% static 'askup/assets/application.js' %}"></script>
+        <link data-turbolinks-track="true" href="{% static 'askup/assets/application.css' %}" media="all" rel="stylesheet">
 
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
-    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-    <meta content="authenticity_token" name="csrf-param">
-    <link href="/static/assets/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon">
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```

Comment:
Why do scheme is not added to the href link?

Reply:
It was specified that way in the original Ruby version website.
![selection_099](https://user-images.githubusercontent.com/2059742/37900507-0b121d68-30f7-11e8-8a9c-2f4aca288ee1.png)

Reply:
That is not the answer on my question